cli/src/actions/database/d1.ts

```diff
@@ -0,0 +1,15 @@
+import { log } from "@clack/prompts";
+
+export function showD1SetupInstructions() {
```

Comment:
TODO - put this in a file and link to it, the text is usually obscured by further cli steps anyhow